modules/nf-core/backsub/tests/main.nf.test

```diff
@@ -0,0 +1,37 @@
+nextflow_process {
+
+    name "Test Process BACKSUB"
+    script "../main.nf"
+    config "./nextflow.config"
+    process "BACKSUB"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "backsub"
+
+
+    test("Should run without failures") {
+        when {
+            process {
+                """
+                // define inputs of the process here. Example:
+                input[0] = [
+                    [ id:'test' ],
+                    file(params.test_data['imaging']['background_subtraction']['image'], checkIfExists: true)
+                ]
+                input[1] = [
+                    [ id:'test' ],
+                    file(params.test_data['imaging']['background_subtraction']['markers'], checkIfExists: true)
+                ]
+                """
+            }
+        }
+        then {
+            assertAll (
+                { assert process.success },
+                { assert process.out.backsub_tif },
```

Comment:
doesn't tif produce a consistent md5?

Reply:
The metadata in the tif changes (the uuid specifically).